packages/ozone/src/api/moderator/listUsers.ts

```diff
@@ -0,0 +1,57 @@
+import { Server } from '../../lexicon'
+import AppContext from '../../context'
+import { ProfileViewDetailed } from '../../lexicon/types/app/bsky/actor/defs'
+import { chunkArray } from '@atproto/common'
+
+// In memory cache for profiles. Current assumption is that the list shouldn't be big enough for this to be an issue in the near term
+const profileCache = new Map<string, ProfileViewDetailed>()
```

Comment:
It would be great to get this cache local to a specific `ctx` rather than free-floating, and perhaps ensure that cache items do naturally get updated over time.  Ideally we would also include these profiles on all `#user` views, not just in this list view.  So maybe we should pull the profile hydration and view logic into `moderatorService`, e.g. adjacent to `view()`.

It also may just not be necessary to cache these at all!  I'd be all for that if it simplifies things.